Backend/AMQPBackendDispatcher.php

```diff
@@ -83,6 +83,10 @@ public function getChannel()
         @trigger_error(sprintf('The method %s is deprecated since version 3.3 and will be removed in 4.0. Use %s::getContext() instead.', __METHOD__, __CLASS__), E_USER_DEPRECATED);
 
         if (!$this->channel) {
+            if (false == $this->context instanceof \Enqueue\AmqpLib\AmqpContext) {
```

Comment:
Can `instanceof` return anything else than a boolean? If not, why are you comparing it with another boolean, in a non-strict way?

Reply:
It is just a matter of visibility.

Reply:
and as far as I know `if (false == ...)` is the equal to `if (!...`

Reply:
replaced it with `!`

Reply:
Thanks for humoring me on this, I always resort to the latter, and I think that's because if you don't do it for `true`, you shouldn't do it for `false`, and you probably don't do it for `true` because you don't say "if true is that the car is green", but simply "if the car is green".